### PR TITLE
Volta project setup

### PR DESCRIPTION
**Pull request by nsklyarov**
# Contribution checklist

## Brief description

This Pull Request updates Node.js version management from NVM to Volta for a more straightforward and dependable setup.

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [x] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (haiodo) [inline package.json:1]: Last time I've was added global package.json, it caused cache issues within rush build's, not sure it is a good idea to add it.

**Comment by haiodo** [comment]
Probable a good idea will be just to have a list of possible node managers and required version of node, platform actually are not strongly depend on particular node version, it could work fine with node starting v14, the one we started with.

File: package.json (hunk 1 above)
Comment (nsklyarov) [reply package.json:1]: Hmm.. I didn’t find any other way to pin the version of node in a project using Volta other than package.json.
But I have found [issue](https://github.com/volta-cli/volta/issues/282) about this.

**Comment by aonnikov** [comment]
> Probable a good idea will be just to have a list of possible node managers and required version of node, platform actually are not strongly depend on particular node version, it could work fine with node starting v14, the one we started with.

Actually I saw a problem in some pod project with node < v18, but don't remember the details already 😄

**Pull request closed without merging**